end-to-end_sales-forecasting/sales_forecasting-end-to-end.ipynb

```diff
@@ -0,0 +1,2701 @@
+{
```

Comment:
<p>Let's remove the appendix from the polished notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='185'/>

Reply:
<p>10-4</p>